code/modules/hydroponics/grown/tomato.dm

```diff
@@ -150,4 +150,6 @@
 			K.visible_message("<span class='notice'>The Killer Tomato growls as it suddenly awakens.</span>")
 			if(user)
 				user.unEquip(src)
+			message_admins("[key_name_admin(usr)] released a killer tomato at ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
```

Comment:
There's a helper for this - https://github.com/ParadiseSS13/Paradise/blob/master/code/__DEFINES/admin.dm#L65

Reply:
I updated it, it uses the helpers now, on the suggestion of someone in coder chat I used user instead of usr, and I added a variable to store the turf in.